### PR TITLE
Update octoprint.markdown

### PR DESCRIPTION
**Pull request by chatainsim**
Add Discovery plugin information

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @chatainsim, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20668).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/octoprint.markdown [hunk 1]
```diff
@@ -23,7 +23,7 @@ ha_platforms:
 
 ### API Key
 
-The Octoprint integration will attempt to register itself via the [application keys plugin](https://docs.octoprint.org/en/master/bundledplugins/appkeys.html). After submitting the configuration UI in Home Assistant, open the Octoprint UI and click allow on the prompt.
+The Octoprint integration will attempt to register itself via the [application keys plugin](https://docs.octoprint.org/en/master/bundledplugins/appkeys.html). After submitting the configuration UI in Home Assistant, open the Octoprint UI and click allow on the prompt. **For this you need the Discovery plugin to be activated in Octoprint**.
```
Comment (frenck) [inline source/_integrations/octoprint.markdown]: I don't think we need to emphasize this in bold. Instead, let's move it to the start of the paragraph and explain that a user should validate the discovery plugin is activated/active/enabled (or whatever Octoprint calls it).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/octoprint.markdown (hunk 1 above)
Comment (chatainsim) [reply source/_integrations/octoprint.markdown]: I've made the change. Thanks

File: source/_integrations/octoprint.markdown [hunk 2]
```diff
@@ -22,8 +22,8 @@ ha_platforms:
 {% include integrations/config_flow.md %}
 
 ### API Key
-
-The Octoprint integration will attempt to register itself via the [application keys plugin](https://docs.octoprint.org/en/master/bundledplugins/appkeys.html). After submitting the configuration UI in Home Assistant, open the Octoprint UI and click allow on the prompt.
+For the integration to work, please check that the plugin Discovery is enabled (you can disable it after integration is done).
```
Comment (frenck) [inline source/_integrations/octoprint.markdown]: That comment is incorrect now, as discovery should be enabled even afterward.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/octoprint.markdown (hunk 2 above)
Comment (chatainsim) [reply source/_integrations/octoprint.markdown]: Sorry, I've missed this information.

**Review by frenck — APPROVED** [review]
Thanks, @chatainsim 👍

**Pull request merged**